wal/watcher.go

```diff
@@ -0,0 +1,587 @@
+// Copyright 2018 The Prometheus Authors
```

Comment:
Year is when the file was created.

Reply:
this file was copied from the prometheus repo so I left he original copyright, I can change the year if you want